.github/workflows/build.yml

```diff
@@ -712,3 +712,11 @@ jobs:
         run: |
           cd packages/e2e-tests
           yarn test:e2e
+
+  job_required_tests:
+    name: All required tests passed or skipped
+    needs: [job_unit_test, job_nextjs_integration_test, job_node_integration_tests]
```

Comment:
Should we make the other jobs also required then? Like playwright, remix and ember? 

I think we shouldn’t block the e2e tests though.

Reply:
Yeah, that's a good question, I also thought about this. I would tend to "yes", esp. when they )(hopefully) also run a bit less often. We can debate about ember and maybe remix I guess, but at the very least the browser integration/playright tests should probably become required as well, I'd say!

Reply:
Agreed, let's try to make them required for now. We can loosen this whenever we need to easily.